test/Microsoft.AspNetCore.Razor.Test/TagHelpers/TagHelperParseTreeRewriterTest.cs

```diff
@@ -14,6 +14,7 @@
 using Microsoft.AspNetCore.Razor.Text;
 using Xunit;
 using Microsoft.AspNetCore.Razor.Parser.TagHelpers.Internal;
+using Microsoft.AspNetCore.Razor.Parser.TagHelpers;
 
 namespace Microsoft.AspNetCore.Razor.Test.TagHelpers
```

Comment:
That's the responsibility of the `TagHelperBlockRewriter`. There's tests in there for all the various quotes.